sdk/src/main/java/com/mapbox/maps/extension/observable/ObservableExtension.kt

```diff
@@ -94,65 +92,65 @@ fun ObservableInterface.unsubscribeMapLoadingError(observer: Observer) =
   unsubscribe(observer, listOf(MapEvents.MAP_LOADING_ERROR))
 
 /**
- * Subscribes an Observer for of event type "map-loading-finished".
+ * Subscribes an Observer for of event type "map-loaded".
  *
  * Observable will hold a strong reference to an Observer instance, therefore,
  * in order to stop receiving notifications, caller must call unsubscribe with an
  * Observer instance used for an initial subscription.
  *
  * @param observer an Observer
  */
-fun ObservableInterface.subscribeMapLoadingFinished(observer: Observer) =
-  subscribe(observer, listOf(MapEvents.MAP_LOADING_FINISHED))
+fun ObservableInterface.subscribeMapLoaded(observer: Observer) =
```

Comment:
nit but
do we have an example somewhere of how to use that extension?
I think may be reasonable adding in some example activity.

Reply:
We have one example in DebugModeActivity, but it doesn't cover all the events. We should make a new example for both the listeners and observer approach.

Reply:
Worth creating a ticket?

Reply:
ticketed at https://github.com/mapbox/mapbox-maps-android/issues/189